tools/privacyidea-token-janitor

```diff
@@ -227,6 +229,8 @@ def _compare_regex_or_equal(_key, given_regex):
 
 def build_tokenvalue_filter(key,
                             tokeninfo_value,
+                            has_not_tokeninfo_key,
+                            has_tokeninfo_key,
```

Comment:
actually we do not need these parameters here since we check it after the database was queried. The filter is only for the database query.